test/backend/lib.fetching.content-service.crowd-tangle-content-service.test.js

```diff
@@ -0,0 +1,105 @@
+var utils = require('./init');
+var expect = require('chai').expect;
+var fs = require('fs');
+var path = require('path');
+var CrowdTangleContentService = require('../../lib/fetching/content-services/crowd-tangle-content-service');
+var ContentService = require('../../lib/fetching/content-service');
+var contentServiceFactory = require('../../lib/fetching/content-service-factory');
+
+
+// Stubs the _httpRequest method of the content service to return the data in the given fixture file.
+function stubWithFixture(fixtureFile, service) {
+
+  // If service is null, creates a CrowdTangleContentService
+  service = service || new CrowdTangleContentService({});
+
+  // Make the stub function return the expected args (err, res, body).
+  fixtureFile = path.join('test', 'backend', 'fixtures', fixtureFile);
+  service._httpRequest = function(params, callback) {
+    callback(null, { statusCode: 200 }, fs.readFileSync(fixtureFile).toString());
+  };
+
+  return service;
+}
+
+describe('CrowdTangle content service', function() {
+
+  it('factory should instantiate correct CrowdTangle content service', function() {
+    var service = new CrowdTangleContentService({});
+    expect(service).to.be.instanceOf(ContentService);
+    expect(service).to.be.instanceOf(CrowdTangleContentService);
+  });
+
+  it('should fetch empty content', function(done) {
+    var service = stubWithFixture('ct-0.json');
+    utils.expectToNotEmitReport(service, done);
+    expect(service._lastReportDate).to.be.undefined;
+    service.once('error', function(err) {
+      done(err);
+    });
+    setTimeout(done, 500);
+  });
+
+  it('should return data from CrowdTangle', function(done) {
+    var service = stubWithFixture('ct-3.json');
+
+    service.once('error', function(err) { done(err); });
+
+    service.on('report', function(reportData) {
+      expect(reportData.metadata.crowdtangleId).to.not.be.undefined;
+    });
+    setTimeout(done, 500);
+  });
+
+
+  // TODO 1
```

Comment:
Why is this TODO here? Can it be removed, or should another test be in its place?

Reply:
Will remove.